train_vnext/training.py

```diff
@@ -158,17 +142,85 @@ def process_post_training_csv(csv_path, training_id, classification_name_to_clas
                 prediction_label = PredictionLabel(training_id, 
                                     int(row[0].split('.')[0]), 
                                     classification_name_to_class_id[class_name], 
-                                    float(row[2]), 
-                                    float(row[3]), 
-                                    float(row[4]), 
-                                    float(row[5]), 
+                                    float(row[2])*float(row[6]), 
+                                    float(row[3])*float(row[6]), 
+                                    float(row[4])*float(row[7]), 
+                                    float(row[5])*float(row[7]), 
                                     int(row[6]), 
                                     int(row[7]), 
                                     float(row[8]), 
                                     float(row[9]))
                 payload_json.append(prediction_label)
     return jsonpickle.encode(payload_json, unpicklable=False)
 
+def save_training_session(config, model_location, perf_location, prediction_labels_location):
+    # First, upload the model to blob storage
+    cur_date_time = "{date:%Y_%m_%d_%H_%M_%S}".format(date=datetime.datetime.now())
```

Comment:
@samiyaakhtar Perhaps we should prefer datetime.utcnow()

Reply:
good catch